test/test_win_evaluation_poker.py

```diff
@@ -0,0 +1,47 @@
+from CardGames import *
+from win_evaluation_poker import *
+from Betting_Class import *
+from River_Class import *
+
+
+def testing():
```

Comment:
Pretty good test it looks like. I'm assuming it passed otherwise it wouldn't be here lol. It covers pretty much all of your code so I like it.